hermes-common/src/main/java/pl/allegro/tech/hermes/common/config/Configs.java

```diff
@@ -16,6 +16,7 @@
     ZOOKEEPER_MAX_RETRIES("zookeeper.max.retries", 29),
     ZOOKEEPER_CONNECTION_TIMEOUT("zookeeper.connection.timeout", 10000),
     ZOOKEEPER_SESSION_TIMEOUT("zookeeper.session.timeout", 10000),
+    ZOOKEEPER_MAX_INFLIGHT_REQUESTS("zookeeper.max.inflight.requests", 1),
```

Comment:
https://cwiki.apache.org/confluence/display/KAFKA/KIP-214%3A+Add+zookeeper.max.in.flight.requests+config+to+the+broker

Reply:
Set it to 1 and it will behave just like the old zookeeper client. We can change this configuration for better performance but for a while we want to have the same behaviour like before this PR.